bfb/infojson.sh

```
@@ -0,0 +1,254 @@
+CONTAINER_IMAGE="localhost/${IMG_NAME}:${RHCOS_VERSION}-latest"
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)